layouts/page/cudf-polars.html

```diff
@@ -0,0 +1,441 @@
+"btn_text" "Get Started"
+"btn_link" "#get-started"
+) }}
+
+<!-- Intro -->
+<section class="row section mb-6">
+  <div class="col">
+    <div class="container">
+
+      <h1 class="text-center mb-10"> BRINGING ACCELERATED COMPUTING TO POLARS <br> [Open Beta] </h1>
+      <div class="row">
+        <div class="col-md-4 py-3">
+          <h2><i class="fa-light fa-file-binary"></i> Zero Code Change</h2>
+          <p>Pass <code>engine=”gpu”</code> to the Polars' collect API to add GPU acceleration to your data processing tasks like data aggregation, merge, window functions, and much more.</p>
+
+          <br>
+
+          <h2 class="mt-4"><i class="fa-sharp fa-light fa-solar-system"></i> Ecosystem Compatible</h2>
+          <p>Polars' growing ecosystem of data visualization, IO, & ML libraries can be accelerated with the GPU engine.</p>
+        </div>
+        <div class="col-md-4 py-3">
+          <h2><i class="fa-solid fa-light fa-microchip"></i> Seamless CPU Fallback</h2>
+          <p>For queries where one or more operations are not supported in cuDF, the entire query execution will gracefully fallback to the default CPU engine.</p>
+
+          <br>
+
+          <h2 class="mt-4"><i class="fa-light fa-bolt-lightning"></i> Query Optimizer</h2>
+          <p>The GPU engine utilizes Polars' query optimizer to ensure efficient execution and minimal memory usage.</p>
+        </div>
+        <div class="col-md-4 py-3">
+          <iframe width="550" height="300" src="https://www.youtube.com/embed/AoKeit2Fbmw?si=Gin-Bwr5ndvaMABI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+        </div>
+      </div>
+
+    </div>
+  </div>
+</section>
+
+<div class="bottom-slant-container">
+  <div class="bottom-slant-down bg-white"></div>
+</div>
+
+<!-- Charts -->
+<section class="row section padding-top-slant">
+  <div class="col">
+    <div class="container">
+
+      <h1 class="text-center mb-4"> Process 100s of Millions of Rows in Seconds </h1>
+      <div class="row">
+        <div class="col-md-6">
+          <h3 class="text-center"> Performance improves as data size grows </h3>
+          <canvas id="perfchartdata" width="500" height="375"></canvas>
+        </div>
+        <div class="col-md-6">
+          <h3 class="text-center"> Accelerate Polars Workflows up to 13x</h3>
+          <canvas id="perfchartspeed" width="500" height="375"></canvas>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <p class="fw-lighter text-center">*GPU: NVIDIA H100, CPU: Intel Xeon W9-3495X (Sapphire Rapids), Storage: Local NVMe derived from TPC-H but these results are not comparable to TPC-H results <a href="https://github.com/pola-rs/polars-benchmark" target="_blank">Learn More <i class="fa-solid fa-arrow-up-right"></i></a></p>
```

Comment:
Can we make this-

PDS-H benchmark | GPU: NVIDIA H100 | CPU: Intel Xeon W9-3495X (Sapphire Rapids) | Storage: Local NVMe <New line>
Note: PDS-H is derived from TPC-H but these results are not comparable to TPC-H results. [Learn More](https://github.com/pola-rs/polars-benchmark)